APIs/schemas/token_schema.json

```
@@ -14,10 +14,17 @@
     },
     "aud": {
       "description": "A JSON array of case-sensitive strings, each containing a StringOrURI value that identifies the recipients that the JWT is intended for",
-      "type": "array",
-      "items": {
-        "type": "string"
-      }
+      "anyOf": [
+        {
+          "type": "array",
+          "items": {
+          "type": "string"
```

indent two more spaces
```suggestion
            "type": "string"
```